vicon_mujoco_init/vicon_transformer.py

```diff
@@ -0,0 +1,188 @@
+import json 
+import os
+from numpy.core.fromnumeric import transpose
+import pytransform3d as tr
+import numpy as np
+from os.path import dirname, abspath, join
+import zmq
+import json
+
+# object names 
+# ['frame_number', 'frame_rate', 'latency', 'my_frame_number', 'num_subjects', 
+# 'on_time', 'subjectNames', 'subject_0', 'subject_1', 'subject_2', 'subject_3', 
+# 'subject_4', 'subject_5', 'subject_6', 'subject_7', 'subject_8', 'subject_9', 
+# 'time_stamp']
+
+
+class ViconJson:
```

Comment:
should really this class be in the demo repo ? Feels like it could be reused on other places ?

Reply:
(if to be reused in other places, I'll do a more detailed review)